cmd2/cmd2.py

```diff
@@ -4039,6 +4042,37 @@ def complete_help_subcommands(
         completer = argparse_completer.DEFAULT_AP_COMPLETER(argparser, self)
         return completer.complete_subcommand_help(text, line, begidx, endidx, arg_tokens['subcommands'])
 
+    def _build_command_info(self) -> tuple[dict[str, list[str]], list[str], list[str], list[str]]:
```

Comment:
Would using a dataclass or `NamedTuple` be more readable than a tuple with 4 elements?  Its fine as is, just thinking.